src/Executor/Values.php

```diff
@@ -94,7 +94,7 @@ public static function getVariableValues(Schema $schema, $definitionNodes, array
      * @return array
      * @throws Error
      */
-    public static function getArgumentValues($def, $node, $variableValues)
+    public static function getArgumentValues($def, $node, $variableValues = null)
```

Comment:
In theory this is possible, but we use this class as a container for pure stateless functions and reference these static methods directly by class name, so it doesn't imply any polimorphism. It is implicitly final. But yeah - we should probably mark such classes as final explicitly.

Anyway it's too late: I released 0.9.4 even before your first comment %) But I am sure it won't break anyone, so let's cross the fingers %)